docs/en-us/vsc-extension/customize-project.md

```diff
@@ -5,7 +5,10 @@
 
 # Customize Projects
 
+> **Note:** If you want to learn more about how to define and structure customizations for Azure DevOps processes – including adding fields, modifying layouts, defining rules, and managing states – please refer to the dedicated [**Process Modification documentation**](/docs/en-us/vsc-extension/process-modification.md).
+
 # [**GitHub (AL-Go)**](#tab/github)
```

Comment:
you can remove the Tab for GitHub and add a note that this is only available for Azure DevOps